acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/AcceptanceTestBase.java

```diff
@@ -36,10 +48,14 @@ protected Node ethNode() {
   @BeforeClass
   public static void setUpBase() {
     Runtime.getRuntime().addShutdownHook(new Thread(AcceptanceTestBase::tearDownBase));
+    setLogLevelToDebug();
```

Comment:
is this useful for all tests, or is a hangover from the failure? (if needed - why not trace?)

Reply:
Removed